new-lamassu-admin/src/pages/Customers/CustomerProfile.js

```diff
@@ -0,0 +1,378 @@
+import { makeStyles } from '@material-ui/core/styles'
```

Comment:
This became a huge component. Do try and create some smaller ones under /pages/Customer/components/ and add whatever you can there. I see a bunch of repetition here. Also feel free to add page specific components there, like the PropertyCard one.